mne/channels/channels.py

```diff
@@ -502,8 +502,9 @@ def plot_sensors(self, kind='topomap', ch_type=None, title=None,
         title : str | None
             Title for the figure. If None (default), equals to ``'Sensor
             positions (%s)' % ch_type``.
-        show_names : bool
-            Whether to display all channel names. Defaults to False.
+        show_names : bool | array of str
```

Comment:
this indeed does what the user requested, but the API is different from plot_topomap here: http://martinos.org/mne/stable/generated/mne.viz.plot_topomap.html. There, it is the `names` argument which takes the list of channel names